modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/management_bar/components/FiltersDropdown.js

```diff
@@ -75,7 +75,12 @@ const FiltersDropdown = () => {
 				</>
 			) : (
 				<ClayDropDown.Group header={Liferay.Language.get('filters')}>
-					<ClayDropDown.Search onChange={onSearch} value={query} />
+					<ClayDropDown.Search
+						aria-label={Liferay.Language.get('input-filter')}
```

Comment:
`input-filter` does not exist in [DXP language keys](https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties). You need to add the key there if you want to use it. But, if you did this, `Input Filter` would be rather technical and non-descriptive label for screen reader to read. Something like `search` exists and would be more descriptive.